numbers/guides/hostedMessaging.md

```diff
@@ -540,6 +657,19 @@ Content-Type: application/xml; charset=utf-8
 </TNs>
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->getInserviceNumbers());
```

Comment:
Same as above, break out to assign as variable

Reply:
done 👍